src/libs/OptionsListUtils.js

```diff
@@ -711,6 +715,36 @@ function getNewChatOptions(
     });
 }
 
+/**
+ * Build the options for the Workspace Member Invite view
+ *
+ * @param {Object} reports
+ * @param {Object} personalDetails
+ * @param {Array<String>} betas
+ * @param {String} searchValue
+ * @param {Array} selectedOptions
+ * @param {Array} excludeLogins
+ * @returns {Object}
+ */
+function getMemberInviteOptions(
+    reports,
```

Comment:
Do we need this argument? We can just remove it and refactor all the usages of this function.

Reply:
Thanks @parasharrajat . I removed report parameter  and updated all  functions. Also removed `selectedOptions`.